debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/QueueSegment.java

```diff
@@ -203,4 +203,23 @@ private void truncateFileAfterOffset(long offset){
         }
 
     }
+
+    // read all events in the queue segemnt and return them as a collection of
+    // strings
+    public String readAllEvents() {
```

Comment:
Are you still using this? Can't find references?

Reply:
Removed